src/Facets/AllBridgeFacet.sol

```diff
@@ -96,13 +129,37 @@ contract AllBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
         ILiFi.BridgeData memory _bridgeData,
         AllBridgeData calldata _allBridgeData
     ) internal {
+        // make sure destinationChainId matches in bridgeData and allBridgeData
+        if (
+            _allBridgeData.destinationChainId !=
+            _getAllBridgeChainId(_bridgeData.destinationChainId)
+        ) revert InvalidCallData();
+
+        // validate receiver address
+        if (_bridgeData.receiver == NON_EVM_ADDRESS) {
+            // destination chain is non-EVM
+            // make sure it's non-zero (we cannot validate further)
+            if (_allBridgeData.recipient == bytes32(0))
+                revert InvalidNonEVMReceiver();
```

Comment:
gj for adding new error, I see the same is in MayanFacet that we can update in the future